.github/actions/build-vsix/action.yml

```diff
@@ -5,7 +5,7 @@ inputs:
   package_command:
      description: 'Package.json package script'
      required: false
-     default: 'package'
+     default: 'package:nls'
```

Comment:
Sorry this is the wrong version AFAIK. You should be modifying the azure-pipelines.prerelease and azure-pipelines.stable.yml

Reply:
This action.yml is used by our build-test.yml